dts/bindings/regulator/regulator-gpio.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO-controlled voltage of regulators
+
+include:
+  - name: base.yaml
+  - name: regulator.yaml
+    property-allowlist:
+      - regulator-name
+      - regulator-min-microvolt
+      - regulator-max-microvolt
+
+compatible: "regulator-gpio"
+
+properties:
+  regulator-name:
+    required: true
+
+  regulator-min-microvolt:
+    required: true
+
+  regulator-max-microvolt:
+    required: true
```

Comment:
these properties can't be required, they just indicate regulator operating range, which may or may not be set by the user.

Reply:
fixed